src/js/components/Anchor/__tests__/Anchor-test.tsx

```diff
@@ -236,10 +236,23 @@ describe('Anchor', () => {
         textDecoration: 'underline',
         fontWeight: 700,
         size: {
```

Comment:
This test might not be the most effective because when it maps through the different sizes it don't create an anchor that would have a gap because the anchors don't have an icon. So we don't actually test the anchor gap.

Reply:
good catch I think Im just missing `small` though 

in the test there is 

```
    <Anchor
          label="Large anchor with icon should receive color on icon"
          icon={<LinkNext />}
          size="large"
        />
```

then 
```
        <Anchor size="medium" label="Anchor with icon" icon={<LinkNext />} />
```

Reply:
☝️  default is `medium`